keyboards/cannonkeys/rekt1800/rekt1800.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+#include "quantum.h"
+#define K_NO KC_NO
+
+#define LAYOUT_all( \
+	K000, K001, K002, K003, K004, K005, K006, K007, K008, K009, K010, K011, K012,       K014, K015, K016, K017, K018, \
+	K100, K101, K102, K103, K104, K105, K106, K107, K108, K109, K110, K111, K112, K113, K114, K115, K116, K117, K118, \
+	K200, K201, K202, K203, K204, K205, K206, K207, K208, K209, K210, K211, K212,       K214, K215, K216, K217, K218, \
+	K300, K301, K302, K303, K304, K305, K306, K307, K308, K309, K310, K311, K312,       K314, K315, K316, K317, K318, \
+	K400, K401, K402, K403, K404, K405, K406, K407, K408, K409, K410, K411, K412,       K414, K415, K416, K417, K418, \
+	K500, K501, K502,                   K506,                   K510, K511, K512,       K514, K515, K516, K517        \
```

Comment:
Your `info.json` file is set up for a split Numpad Enter, but there's no argument here for it. The JSON file seems to indicate that there should be a `K518` here.

https://github.com/qmk/qmk_firmware/blob/32e247653fa9e3e32fc6319314aa9324b8c01912/keyboards/cannonkeys/rekt1800/rekt1800.h#L12

```suggestion
	K500, K501, K502,                   K506,                   K510, K511, K512,       K514, K515, K516, K517, K518  \
```

Both keymaps would need to be updated for this as well.

Reply:
There is actually no split numpad enter on the PCB. I will have to fix the JSON